src/lib/freebsd/Makefile

```diff
@@ -0,0 +1,41 @@
+deps = gdk-2.0 gdk-pixbuf-2.0 atk
```

Comment:
Everything in src/lib/ is currently experimental. So src/lib/freebsd is not necessary.